data/json/proficiencies/traps.json

```diff
@@ -28,7 +28,8 @@
     "can_learn": true,
     "default_time_multiplier": 2,
     "default_fail_multiplier": 1.25,
-    "time_to_learn": "10 h"
+    "time_to_learn": "10 h",
+    "required_proficiencies": [ "prof_lockpicking" ]
```

Comment:
These aren't related skills

Reply:
Some safes have non-combination locks - as in, can be opened via picking.

Reply:
If we have pickable safes in game, they should use the lockpicking proficiency. Opening a combination lock using a stethoscope just isn't related to picking a lock with a tension wrench, and any shared knowledge is covered by the fact that they both use the Devices skill.